examples/classification/nncf_colab_demo.ipynb

```diff
@@ -0,0 +1 @@
"                        0.1,\r\n","                        0.2,\r\n","                        0.3,\r\n","                        0.4,\r\n","                        0.5,\r\n","                        0.6\r\n","                    ],\r\n","                }\r\n","            }\r\n","    }\r\n","\r\n","    # create json files, that will be used by nncf later\r\n","    for config_key, config_val in configs.items():\r\n","        write_json(config_val, config_key)\r\n","    \r\n","    return configs"],"execution_count":11,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"9d8LOmKut36x"},"source":["## Train/tune"]},{"cell_type":"code","metadata":{"id":"qk0Nub3KiqZK"},"source":["params = {}\n","\n","params['arch'] = 'resnet18'\n","params['num_classes'] = 200 # 200 is for tiny-imagenet, default is 1000 for imagenet\n","params['data'] = 'tiny-imagenet-200' # path to data\n","\n","params['init_lr'] = 1e-4\n","params['batch_size'] = 256\n","params['workers'] = 4\n","params['start_epoch'] = 0 # updated automatically if training is resumed\n","params['epochs'] = 15 # 15 is for full precision training, will be updated (increased) in case of tuning with nncf \n","params['pretrained'] = True # pretrained model on Imagenet\n","\n","params['resume'] = PATH + 'model_best.pth.tar'  # path to latest checkpoint (or None)\n","params['checkpoint_compressed'] = False\n","\n","params['evaluate'] = False\n","\n","params['use_nncf'] = True\n","\n","if params['use_nncf']:\n","    # create all config files\n","    configs = create_json_files(params['batch_size'], input_size=224)\n","\n","    # choose config file\n","    algorithm_config = 'quantization.json'\n","    params['nncf_config_file'] = 'config_files/' + algorithm_config\n","\n","    # update tune params to fit certain compression algorithm\n","    params['init_lr'] = configs[algorithm_config]['optimizer']['base_lr']\n","    params['epochs'] = params['epochs'] + configs[algorithm_config]['epochs']\n","\n","\n","# Run certain algorithm once\n","main(params)\n","\n","# Iterate over algorithms\n","algorithm_configs = ['quantization.json', 'pruning.json', 'sparsity.json']\n","for algorithm_config in algorithm_configs:\n","    # Run the tuning procedure:\n","    # print(algorithm_config)\n","    # params['nncf_config_file'] = 'config_files/' + algorithm_config\n","    # update tune params\n","    # main(params)\n","    pass"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"ENAbqFpdWSlE"},"source":["# Export to ONNX that is supported by the OpenVINO™ toolkit\r\n","compression_ctrl.export_model(\"model_compressed.onnx\")"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"_I_G-g9TPWkl"},"source":["#Export to OpenVINO™ Intermediate Representation (IR)\r\n","To export a model to the OpenVINO IR and run it using the Intel® Deep Learning Deployment Toolkit, refer to this [tutorial](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html)."]}]}
```

Comment:
Does this look right? Does every `.ipynb` file necessarily have to consist of a single line, or are other formatting options available?

Reply:
Probably yes, you can check how it is rendered by GitHub [here](https://github.com/negvet/nncf/blob/et/colab/examples/classification/nncf_colab_demo.ipynb), refresh if not able to load from the first time.

Actually, I just downloaded the `.ipynb` file from Google Colaboratory.

Another option is just to provide a link to read-only Colab in classification readme. It is common practice - no need to manually upload it to colaboratory. The example can be seen [here](https://github.com/yandexdataschool/Practical_DL/blob/spring2019/week01_backprop/README.md) in Practice section.